tests/backstop/engine_scripts/playwright/onReadyResize.js

```diff
@@ -0,0 +1,6 @@
+module.exports = async (page, scenario) => {
+  await require('./onReady')(page, scenario);
+
+  await page.evaluate(() => window.dispatchEvent(new Event('resize')));
+  await page.waitForTimeout(300);
```

Comment:
does this actually need to be increased to 300ms or can we reduce this? It looks like the resize debounce is 100ms and there is a 200ms delay in responding to page orientation. Since it's used in a few tests it would be a shame to add to the runtime.